authorization/jsonld-contexts/authorization.jsonld

```diff
@@ -0,0 +1,11 @@
+{
+	"@context": {
+		"name": "https://ontology.eglobalmark.com/authentication#name",
```

Comment:
you should also adapt the namespace to be consistent (i.e. change the `authentication` in it)